package.json

```diff
@@ -15,7 +15,9 @@
     "Sam Kummary <sam@modusbox.com>",
     "Steven Oderayi <steven.oderayi@modusbox.com>",
     "Valentin Genev <valentin.genev@modusbox.com>",
-    "Shashikant Hirugade <shashikant.hirugade@modusbox.com>"
+    "Shashikant Hirugade <shashikant.hirugade@modusbox.com>",
+    "Paweł Marzec <pawel.marzec@modusbox.com>",
+    "Kevin Leyow <kevin.leyow@modusbox.com>"
```

Comment:
what about bumping the version?

Reply:
I was hoping to have https://github.com/mojaloop/central-ledger/pull/818 to do it for me but now that I think of it, it wouldn't catch this `feat` in the bump.

done and ty.